distr_tracing/distr_tracing_install/distr-tracing-deploying-otel.adoc

```diff
@@ -9,3 +9,10 @@ toc::[]
 The {OTELName} Operator uses a custom resource definition (CRD) file that defines the architecture and configuration settings to be used when creating and deploying the {OTELName} resources. You can either install the default configuration or modify the file to better suit your business requirements. 
 
 include::modules/distr-tracing-config-otel-collector.adoc[leveloffset=+1]
+
+include::modules/distr-tracing-config-otel-send-metrics-monitoring-stack.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_deploy-otel"]
+== Additional resources
+* xref:../../monitoring/enabling-monitoring-for-user-defined-projects.adoc[Enable monitoring stack for user defined projects.]
```

Comment:
I've checked other pages in the docs, and a period is not used to end a link under Additional resources:
```suggestion
* xref:../../monitoring/enabling-monitoring-for-user-defined-projects.adoc[Enable monitoring stack for user defined projects]
```